layouts/_partials/endpoints-toc.html

```diff
@@ -0,0 +1,42 @@
+{{/* Minimal list of API endpoints for the current page. */}}
```

Comment:
it might be nice to document here, that this expects some values in `Scratch`, and what format it expects.